src/main/java/com/odysseusinc/arachne/datanode/service/messaging/CohortCharacterizationRequestHandler.java

```diff
@@ -99,8 +100,9 @@ public CohortCharacterizationRequestHandler(AtlasClient atlasClient,
             if (Objects.nonNull(definition)) {
                 String content = sqlRenderService.renderSql(definition);
                 if (Objects.nonNull(content)) {
-                    String cohortSqlFileName = definition.getName().trim() + CommonFileUtils.SQL_EXT;
-                    files.add(new MockMultipartFile(cohortSqlFileName, content.getBytes()));
+                    String cohortSqlFileName = definition.getName().trim() + CommonFileUtils.OHDSI_SQL_EXT;
+                    String cohortSql = CohortService.IGNORE_PREPROCESSING_MARK + "\r\n" + content;
```

Comment:
Let's move it some separate method, cause it's already used in a couple of places